packages/core/core/src/index.js

```diff
@@ -18,3 +18,5 @@ export {
   INTERNAL_RESOLVE,
   INTERNAL_TRANSFORM,
 } from './Parcel';
+
+export * from './parcel-v3';
```

Comment:
Is this okay? I don't know if it can be conditionally included

Reply:
Why do we need to export this from core?